Automattic-Tracks-iOS.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'Automattic-Tracks-iOS'
-  s.version       = '0.10.0'
+  s.version       = '0.10.1-beta.1'
```

Comment:
When the PR is about to get merged we increase the podpsec version, and as soon as it's merged we do a new release using this version number.  This ensures the new version is published in CocoaPods for other Apps to use.

Reply:
We're SO COOL that we're using SPM @diegoreymendez 😎